lncrawl/sources/babelnovel.py

```diff
@@ -119,35 +116,11 @@ def parse_chapter_item(self, list_url):
         return chapters
     # end def
 
-    def parse_content_css(self, url):
-        try:
-            soup = self.get_soup(url)
-            content = re.findall(
-                'window.__STATE__ = "([^"]+)"', str(soup), re.MULTILINE)
-            data = json.loads(unquote(content[0]))
-            cssUrl = self.absolute_url(data['chapterDetailStore']['cssUrl'])
-            logger.info('Getting %s', cssUrl)
-            css = self.get_response(cssUrl).text
-            baddies = css.split('\n')[-1].split('{')[0].strip()
-            self.bad_selectors = baddies
-            logger.info('Bad selectors: %s', self.bad_selectors)
-        except:
-            self.bad_selectors = []
-            logger.exception('Fail to get bad selectors')
-        # end for
-    # end def
-
     def download_chapter_body(self, chapter):
         logger.info('Visiting %s', chapter['json_url'])
         data = self.get_json(chapter['json_url'])
 
         soup = BeautifulSoup(data['data']['content'], 'lxml')
-        if self.bad_selectors:
-            for tag in soup.select(self.bad_selectors):
-                tag.extract()
```

Comment:
babel novel puts some jumbled text along with main chapter content, which are hidden using CSS. if you remove this, the jumbled text will be visible in the epub.

Reply:
I don't think they do that anymore. I downloaded 10 chapters of random novel and didn't see any weird strings. Then I browsed the HTML of a different one and there were no strange things in the chapter content HTML. Are you sure it's still a thing? As it is it doesn't work either way, since the regex doesn't match anything and the whole thing just throws an exception.

Reply:
yeah, you are right. i've checked several novels now, it doesn't do that anymore.